### PR TITLE
glm: fix license, copy cmake config files

### PR DESCRIPTION
**Pull request by Radfordhound**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/glm/template [hunk 1]
```diff
@@ -30,6 +32,6 @@ post_install() {
 		dos2unix "$f"
 	done
 	rm -f "${DESTDIR}/usr/include/CMakeLists.txt"
-	sed -n '88,110p' manual.md > LICENSE
+	mv copying.txt LICENSE
 	vlicense LICENSE
```
Comment (tibequadorian) [inline srcpkgs/glm/template:36]: use `vlicense` to install the license, the 2nd argument can be used to change the filename

Comment (Radfordhound) [reply srcpkgs/glm/template]: My bad, somehow missed that when I looked in the packages manual. Should be fixed now

**Comment by Radfordhound** [comment]
I've force pushed to remove the lengthy descriptions from my commit messages so the lint template check succeeds.

File: srcpkgs/glm/template [hunk 2]
```diff
@@ -30,6 +32,5 @@ post_install() {
 		dos2unix "$f"
 	done
 	rm -f "${DESTDIR}/usr/include/CMakeLists.txt"
```
Comment (tibequadorian) [inline srcpkgs/glm/template:34]: I think they meant
```suggestion
	rm -f "${DESTDIR}/usr/include/glm/CMakeLists.txt"
```

**Comment by tibequadorian** [comment]
Don't seperate your changes into multiple commits. Just do one commit with a description. You also need to revbump.

File: srcpkgs/glm/template (hunk 2 above)
Comment (Radfordhound) [reply srcpkgs/glm/template]: Yeah that seems to be the path that was intended. I've added this change to the PR

**Comment by Radfordhound** [comment]
Gotcha. I've merged all the commits into one and bumped the revision number to 2.

**Review by tibequadorian — APPROVED** [review]
Thank you

**Comment by Radfordhound** [comment]
Bumping this as I believe it to still be relevant.

**Comment by Radfordhound** [comment]
Not trying to nag lol, but bumping this again as I still believe it to be relevant

File: srcpkgs/glm/template [hunk 3]
```diff
@@ -29,7 +31,6 @@ post_install() {
 	for f in $(find ${DESTDIR}/usr/include -type f); do
 		dos2unix "$f"
 	done
-	rm -f "${DESTDIR}/usr/include/CMakeLists.txt"
-	sed -n '88,110p' manual.md > LICENSE
-	vlicense LICENSE
+	rm -f "${DESTDIR}/usr/include/glm/CMakeLists.txt"
```
Comment (classabbyamp) [inline srcpkgs/glm/template:34]:
```suggestion
	rm "${DESTDIR}/usr/include/glm/CMakeLists.txt"
```

**Review by classabbyamp — COMMENTED** [review]
can you rebase and fix conflicts?

**Comment by Radfordhound** [comment]
Sure thing. Conflicts should be resolved now, and I've removed the -f from the remove command as suggested.

**Pull request closed without merging**